WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
It requires a certain Moral Authority for the religious head to whine about heresy? Also, what is stopping him from sending letters requesting a cleanup against heresy? That's cheap and easy to do.